_data/mentors.yml

```diff
@@ -174,3 +174,9 @@
   image: JasonWong.jpg
   twitter: codethejason
   bio: A cryptocurrency enthusiast.
+  
+- name: Hon Nguyen
+  github: vanhonit
+  image: hon.png
+  twitter: vanhonit
+  bio: works on linux server and management.
```

Comment:
I saw it on his linked in page. I am so sorry for getting you angry by the way, i was so frustrated with family issues.